api-samples/webRequest/background.js

```diff
@@ -0,0 +1,10 @@
+// Copyright 2021 Google LLC
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file or at
+// https://developers.google.com/open-source/licenses/bsd
```

Comment:
We should use the updated Apache licence here, see https://github.com/GoogleChrome/chrome-extensions-samples/blob/main/api-samples/scripting/index.js :)